me.html

```diff
@@ -0,0 +1,142 @@
+      <a href="#"><img src="./images/facebook.png"  width ="10" height="10px" alt="logo"></a>
+      <a href="#"><img src="./images/twitter.png"  width ="10" height="10px" alt="logo"></a>
+      <a class="social-items" href="#">English</a>
+      <a class="social-items"  href="#">My page</a>
+      <a class="social-items"  href="#">logout</a>
+    </div>
+    <div class="nav-item-desk">
+     <a  href="index.html"><img src="./images/ralph.jpg"  width ="78" height="30px" alt="logo"></a>
+      <div class="nav-items">
+        <a class="nav-link1" href="me.html">About</a>
+      <a class="nav-link" href="#program">Programs</a>
+      <a class="nav-link" href="#contactt">Join</a>
+      <a class="nav-link" href="#contactt">Sponsor</a>
+      <a class="nav-link" href="#contactt">News</a>
+      <a class="nav-link6" href="#contactt">CC Campaign</a>
+    </div>
+    </div>
+    <div id="nav-bar">
+      <div class="nav-item">
+          <a class="nav-link" href="#about">About</a>
+          <a class="nav-link" href="#program">Programs</a>
+          <a class="nav-link" href="#contactt">Join</a>
+          <a class="nav-link" href="#contactt">Sponsor</a>
+          <a class="nav-link" href="#contactt">News</a>
+          <a class="nav-link" href="#contactt">CC Campaign</a>
+      </div>
+      <div id="ham" class="hamburger">
+        <div class="hamburger-bar"></div>
+        <div class="hamburger-bar"></div>
+        <div class="hamburger-bar"></div>
+      </div>
+   </nav>
+  </header>
+  <body>
+    <main>
+      <section id ="about" class="third-sction">
+        <h3 class="hello1">"Hello everyone"</h3>
+        <h1 class="ralph2">RALPH IS STUDY PLATFORM <br>YOU CAN ACCESS </h1>
+            
+        <p class="approach">
+          By far the most popular approach today is online learning. According to the Sloan Consortium,
+          online enrollments continue to grow at rates faster than for the broader student population and
+          institutes of higher education expect the rate of growth to continue increasing. Some of the key
+          findings: 
+        </p>
+        <p class="aps-deck">
+          By far the most popular approach today is online learning. According to the Sloan Consortium,
+          online enrollments continue to grow at rates faster than for the broader student population and
+          institutes of higher education expect the rate of growth to continue increasing. Some of the key
+          findings: By far the most popular approach today is online learning. According to the Sloan Consortium,
+          online enrollments continue to grow at rates faster than for the broader student population and
+          institutes of higher education expect the rate of growth to continue increasing. Some of the key
+          findings: 
+        </p>
+        <P class="reach">
+          Please contact us about Ralph03 online study via email <span id="email">ekpenisiraphael@gmail.com</span>
+        </P>
```

Comment:
- [ Optional ] Kindly work on the "About " page, Your Heading are not fully responsive for Desktop Screen size, Please try to adjust it to make Your website Fully Responsive. 

Your About Page Look Likes this  : 
![Capture1](https://user-images.githubusercontent.com/93598189/192115978-7297511f-a272-4b5a-b89a-4f48f753df2d.PNG)

The Design About Page Look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/192115998-c145d87f-f41a-40eb-90cd-e9feaf804d23.PNG)